assets/scss/base/layouts/_page.scss

```diff
@@ -0,0 +1,325 @@
+/*
+Basic
+
+The basic page layout containing a BETA banner, the service info banner with HMRC logo and example body content.
+
+Markup:
+<main id="wrapper" role="main">
+  <div class="centered-content">
+    <div class="service-info">
+      <div class="beta-banner">
+        <p>
+          <strong class="phase-tag">BETA</strong>
+            <span>
+              This is a new service. Help us improve it - send your
+              <a id="feedback-link" href="#">feedback</a>.
+            </span>
+        </p>
+      </div>
+      <div class="logo">
+        <span class="organisation-logo organisation-logo-medium">
+          HM Revenue &amp; Customs
+        </span>
+      </div>
+    </div>
+  </div>
+  <div id="content">
+    <article class="content__body">
+      <header>
+        <h1>Page Title</h1>
+      </header>
+      <section>
+        <p>Page content</p>
+      </section>
+    </article>
+  </div>
+</main>
+
+Discussion: https://hmrcdigital.hackpad.com/Layouts-H4b5EgRDPYL
+
+Styleguide Layouts.Basic
+*/
+
+
+/*
+Sidebar
+
+The basic page layout containing the service info banner with HMRC logo and example body content. This layout
+demonstrates the use of `.subsection` and `.subsection--wide` styles with example body content and a sidebar with
+example content demonstrating the `.section` styles available.
+
+Markup:
+<main id="wrapper" role="main">
+  <div class="centered-content">
+    <div class="service-info">
+      <div class="logo">
+        <span class="organisation-logo organisation-logo-medium">
+          HM Revenue &amp; Customs
+        </span>
+      </div>
+    </div>
+  </div>
+  <div id="content">
+    <article class="content__body">
+      <header>
+        <h1>Page Title</h1>
+      </header>
+      <section>
+        <p>Page content</p>
+      </section>
+      <section class="subsection">
+        <h2>Subsection Title</h2>
+        <p>Subsection content</p>
+      </section>
+      <section class="subsection--wide">
+        <h2>Wide Subsection Title</h2>
+        <p>Wide Subsection content</p>
+      </section>
+    </article>
+    <aside class="sidebar">
+      <section class="section section--blue-top">
+        <h2>Section Title</h2>
+        <p>Section content</p>
+      </section>
+      <section class="section section--branded">
+        <h2>Section Title</h2>
+        <p>Section content</p>
+      </section>
+    </aside>
+  </div>
+</main>
+
+Styleguide Layouts.Sidebar
+*/
+
+
+/*
+Hero
```

Comment:
You've confused this file with a "Layout" bucket. That's future work so just focus on documenting the file for this PR